src/Resources/config/routing/admin/ajax/brand.yml

```diff
@@ -0,0 +1,26 @@
+loevgaard_sylius_brand_admin_ajax_brands_by_phrase:
+    path: /search
+    methods: [GET]
+    defaults:
+        _controller: loevgaard_sylius_brand.controller.brand:indexAction
+        _format: json
+        _sylius:
+            serialization_groups: [Autocomplete]
```

Comment:
I haven't used this autocomplete feature before, but what's up with the upper case `A`? :D

Reply:
Sylius use convention `Default`, `Detailed`, `Autocomplete` and few more serialization groups.
`Default` is for `index` endpoints, `Detailed` is for `show`, `create`, `update` endpoints (see `ResourceLoader` for more info), `Autocomplete` is for AJAX requests used to autocomplete form elements (`SomethingAutocompleteChoiceType`).

Its just convention.

Reply:
Okay :D